src/components/Carousel.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Carousel = props => {
+  return typeof props.images === 'object' && props.images.length ? (
+    <>
```

Comment:
typo? new syntax i'm unaware of? looks like a fragment...?

Reply:
yep, that's just a short version of a fragment:
https://reactjs.org/docs/fragments.html#short-syntax

Reply:
see https://reactjs.org/docs/fragments.html#short-syntax

motivation is we don't want to put extra nodes in the dom tree (`<div>`s) if we don't actually need them semantically.

Reply:
👏  thanks guys